packages/filesystem/src/browser/file-tree/file-tree-model.ts

```diff
@@ -86,13 +86,16 @@ export class FileTreeModel extends TreeModelImpl implements LocationService {
         if (this.isFileContentChanged(change)) {
             return;
         }
-        const parent = this.getNode(change.uri.parent.toString());
-        if (DirNode.is(parent) && parent.expanded) {
-            accept(parent);
+        const parentNodes = this.getNodes((node: FileStatNode) => change.uri.parent.toString() === node.uri.toString());
```

Comment:
Is there a way to look up by an id still? An idea was to have fast look up, now it seems to travers the whole tree to find a node.